backend/src/main/java/org/cryptomator/hub/license/RandomMinuteSleeper.java

```diff
@@ -0,0 +1,17 @@
+package org.cryptomator.hub.license;
+
+import jakarta.enterprise.context.ApplicationScoped;
+
+import java.util.Random;
+
+@ApplicationScoped
+public class RandomMinuteSleeper {
```

Comment:
I wonder if there is a better name for it because I wouldn't understand that it would sleep between 0 and 60 minutes. But this shouldn't be a blocker, just wanted to mention it. We don't want to delay the release because of this.

Reply:
I don't think the name is perfect either, but I couldn't find a better one yesterday and just now. I'm open to suggestions :)

Reply:
Maybe something like `RandomIntervalSleeper`? But then you would have to add a param to `sleep()` like `maxMinutesInterval`. Not sure if this is better.